src/app/beer_garden/db/mongo/util.py

```diff
@@ -257,13 +257,23 @@ def ensure_v3_27_model_migration():
         db.drop_collection("legacy_role")
 
 
+def ensure_v3_29_model_migration():
+    db = get_db()
+    if not contains_field("request", ["command_display_name"]):
+        request_collection = db.get_collection("request")
```

Comment:
Add Warning statement like the other migration scripts to let the Admins know you are updating their database.